src/services/services.ts

```diff
@@ -3987,7 +3987,7 @@ module ts {
 

```

Comment:
instead, just call normalizeSlashes up here and store the result into fileName.  Then you can use that fileName as written in the below code to keep things simple.